nebula-spark-connector/README.md

```diff
@@ -74,11 +73,44 @@ Nebula Spark Connector 2.0 依赖 Nebula Java Client 2.0。
       .build()
     val vertex = spark.read.nebula(config, nebulaReadVertexConfig).loadVerticesToDF()
   ```
-更多使用示例请参考 [Example](https://github.com/vesoft-inc/nebula-spark-utils/tree/master/example/src/main/scala/com/vesoft/nebula/examples/connector) 。
-## 贡献
 
-Nebula Spark Connector 2.0 是一个完全开源的项目，欢迎开源爱好者通过以下方式参与：
+  读取 Nebula Graph 的点边数据构造 Graphx 的图：
+  ```
+    val config = NebulaConnectionConfig
+      .builder()
+      .withMetaAddress("127.0.0.1:45500")
+      .build()
+    val nebulaReadVertexConfig = ReadNebulaConfig
+      .builder()
+      .withSpace("exchange")
+      .withLabel("person")
+      .withNoColumn(false)
+      .withReturnCols(List("birthday"))
+      .withLimit(10)
+      .withPartitionNum(10)
+      .build()
+    val nebulaReadEdgeConfig = ReadNebulaConfig
+      .builder()
+      .withSpace("exchange")
+      .withLabel("knows1")
+      .withNoColumn(false)
+      .withReturnCols(List("timep"))
+      .withLimit(10)
+      .withPartitionNum(10)
+      .build()
+
+    val vertex = spark.read.nebula(config, nebulaReadVertexConfig).loadVerticesToGraphx()
+    val edgeRDD = spark.read.nebula(config, nebulaReadEdgeConfig).loadEdgesToGraphx()
+    val graph = Graph(vertexRDD, edgeRDD)
+  ```
+  After getting Graphx's Graph, you can develop graph algorithms in Graphx like [Nebula-Spark-Algorithm](https://github.com/vesoft-inc/nebula-java/tree/v1.0/tools/nebula-algorithm).
+
+For more use details please refer to [Example](https://github.com/vesoft-inc/nebula-spark-utils/tree/master/example/src/main/scala/com/vesoft/nebula/examples/connector).
+
+## How to Contribute
+
+Nebula Spark Connector 2.0 is a completely opensource project, opensource enthusiasts are welcome to participate in the following ways:
 
-- 前往 [Nebula Graph 论坛](https://discuss.nebula-graph.com.cn/ "点击前往“Nebula Graph 论坛") 上参与 Issue 讨论，如答疑、提供想法或者报告无法解决的问题
-- 撰写或改进文档
-- 提交优化代码
+- Go to [Nebula Graph Forum](https://discuss.nebula-graph.com.cn/ "go to“Nebula Graph Forum") to participate in issue discussions, such as answering questions, providing ideas, or reporting unsolvable problems;
```

Comment:
Go to [Nebula Graph Forum](https://discuss.nebula-graph.com.cn/ "go to“Nebula Graph Forum") for help.

Reply:
Maybe users can also answer questions and propose new requirements.

Reply:
Go to Nebula Graph Forum to discuss with other users. You can raise your own questions, help others, share your thoughts.

Reply:
got it! Thanks